scripts/python/docs-automation/sync_dotnet_tocs.py

```diff
@@ -0,0 +1,224 @@
+# This script is intended for use in intermediate doc repos generated from docs.ms CI.
```

Comment:
Do we copy this script over to the docs CI pipeline? Any reason this script isn't in their repo if thatis the case?

Reply:
This is what we have originally for other three repos: https://github.com/Azure/azure-sdk-tools/blob/main/scripts/python/docs-automation/sync_tocs.py
 
The only difference between the two scripts are the inputs. sync_tocs.py takes yaml as input and sync_dotnet_tocs.py uses xml as input.

I just followed the convention we have. I agreed with you that docs repo might be the best place to have the script.